StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Elements/CardSection/CardSectionElement.swift

```diff
@@ -146,22 +147,6 @@ final class CardSectionElement: ContainerElement {
         self.preferredNetworks = preferredNetworks
         self.lastPanElementValidationState = panElement.validationState
         cardSection.delegate = self
-
-        // Hook up CBC analytics after self is initialized
-        self.cardBrandDropDown?.didPresent = { [weak self] in
```

Comment:
Is didPresent used anywhere now? We may be able to remove it, same w/ didTapClose.

Reply:
Removed!